src/js/common/schemaform/definitions/address.js

```diff
@@ -48,7 +49,7 @@ export function schema(currentSchema, isRequired = false) {
   };
 }
 
-export function uiSchema(label = 'Address', useStreet3 = false) {
+export function uiSchema(label = 'Address', useStreet3 = false, isRequired = null) {
```

Comment:
Looks like `isRequired` takes a function? Can we put that in a docstring up here?